src/gbd_mapping_generator/data.py

```diff
@@ -322,12 +322,17 @@ def get_coverage_gap_list():
 
 def get_coverage_gap_data():
     out = _SPECIAL_COVERAGE_GAPS
-
     for c in gbd.get_coverage_gap_list():
-        metadata = get_coverage_gap_metadata(c)['exposure']
-        restrictions = tuple((k, v) for k, v in metadata['restrictions'].items())
-        levels = tuple((k, v) for k, v in metadata['levels'].items())
-        out.append((c, None, metadata['distribution'], restrictions, levels,
-                    metadata['affected_causes'], metadata['affected_risks']))
+        metadata = get_coverage_gap_metadata(c)
```

Comment:
`get_coverage_gap_metadata should return a flat dictionary without the top level `exposure` and `relative_risk` keys.  

Having those top level keys makes this code much harder to read and understand.